### DISCUSSION TITLE
IM run in quiet mode

### DISCUSSION THREAD

**Discussion by njsgastro** [ImageMagick/ImageMagick#6510] [General]
I'm incorporating a call to IM from a VB.net application to crop an image . Each time IM runs it pops open a DOS window. Is it possible to have IM run silently  so it doesn't pop up that dos window? I'm on Windows 10  and  latest IM which I loaded the other day.

Thanks 
 Nor

**Comment by snibgo** [comment]
This seems to be a question about VB.net, not about ImageMagick. (The "dos window" is created by VB.net, not IM.)

VB.net has many facilities that might do what you want, such as the ProcessStartInfo.CreateNoWindow property.

**Comment by dlemstra** [comment]
You might also want to consider to use Magick.NET. You then also dont need to install ImageMagick.
